symbols/overlay29.yml

```diff
@@ -8500,6 +8500,44 @@ overlay29:
         Called when the in-dungeon "others" menu is open. Does not return until the menu is closed.
         
         return: Always 0
+    - name: GetAiUseItemProbability
+      address:
+        EU: 0x231EAC4
+        NA: 0x231DEB8
+        JP: 0x231F138
+      description: |-
+        Called to get the probability of an item being used or thrown by an AI on the current turn.
+
+        r0: Pointer to either the user if it is an item used by the AI or the target if it is an item thrown by the AI
+        r1: Pointer to item
+        r2: Unknown 2-bit bitvector. Bit 1 influences only the Warp Seed case, while Bit 0 influences various items.
+        
+        return: Integer in range [0, 100]
+      - name: IsAdjacentToEnemy
+      address:
+        EU: 0x231F358
+        NA: 0x231E74C
+        JP: 0x231F9CC
+      description: |-
+        Called to check if a hostile entity is present in any of the tiles adjacent to an entity.
+
+        r0: Pointer to entity
+        
+        return: True if yes, false if no
+      - name: GetRelationBetweenMonsters
+      address:
+        EU: 0x2302188
+        NA: 0x230157C
+        JP: 0x23027FC
+      description: |-
+        Called to check if two monsters are allies, enemies, or neither (one is a shopkeeper, decoy, etc.)
+
+        r0: Pointer to entity 1
+        r1: Pointer to entity 2
+        r2: ? (If == 0, will make function return 2 if r1 is invisible and r0 can see invisible monsters)
```

Comment:
r2 is true if entity 1 should not take into account the target's invisibility (i.e., target them anyway even when invisible).

r3 is true if entity 1 should ignore a petrified target. This is used to ensure that your allies don't attack petrified targets, since attacking a petrified target will un-petrify them.

Reply:
So uint8_t freeze in statuses is 6 if the Pokémon is petrified? I suspected as much but it's weird that isn't documented, it only says that it 1 means a freeze status condition.

Reply:
The statuses are grouped into a number of different categories such that you can only be inflicted with one status out of each category at a time. The frozen field encompasses several different statuses like freeze, wrap, ingrain, and petrified, which are indicated by different values on the frozen field. Petrified corresponds to 6.

Reply:
Is there any documentation for what values correspond to what?

Reply:
Never mind I found it